sdk/storage/azure-storage-blob-changefeed/pom.xml

```diff
@@ -47,6 +47,8 @@
 
       --add-reads com.azure.storage.blob.changefeed=com.azure.http.netty
       --add-reads com.azure.storage.blob.changefeed=com.azure.core.http.okhttp
+      --add-reads com.azure.storage.blob.changefeed=com.azure.core.http.jdk.httpclient
+      --add-reads com.azure.storage.blob.changefeed=com.azure.core.http.vertx
```

Comment:
Do we need these add-reads?

If adds the readability edge from blob.changefeed to module `http.jdk.httpclient` and `http.vertx`.

I tested in the ACS-Identity with only add the jdk and vertx dependencies like below line 146-151 and 190-206. It passed the live test pipeline.

Reply:
Storage library needs them as they interact with the HTTP clients directly.